src/integrations/ga/GaToSplit.js

```diff
@@ -1,7 +1,257 @@
+    const suffix = model.get(mapping.eventTypeIdSuffix[hitType]);
+    if (suffix)
+      eventTypeId += '.' + suffix;
+
+    const value = model.get(mapping.eventValue[hitType]);
+
+    const properties = {};
+    const fields = mapping.eventProperties[hitType];
+    if (fields) {
+      for (let i = 0; i < fields.length; i++) {
+        properties[fields[i]] = model.get(fields[i]);
+      }
+    }
+
+    return {
+      eventTypeId,
+      value,
+      properties,
+    };
+  };
+}
+
+export const defaultMapper = mapperBuilder(defaultMapping);
+
+/**
+ * Return a new list of identities removing invalid and duplicated ones.
+ *
+ * @param {Array} identities list of identities
+ * @returns list of valid and unique identities, or undefined if `identities` is not an array.
+ */
+export function validateIdentities(identities) {
+  if (!Array.isArray(identities))
+    return undefined;
+
+  // Remove duplicated identities
+  const uniqueIdentities = uniqO(identities);
+
+  // Filter based on rum-agent identities validator
+  return uniqueIdentities.filter(identity => {
+    if (!identity)
+      return false;
+
+    const maybeKey = identity.key;
+    const maybeTT = identity.trafficType;
+
+    if (!isString(maybeKey) && !isNumber(maybeKey))
+      return false;
+    if (!isString(maybeTT))
+      return false;
+
+    return true;
+  });
+}
+
+/**
+ * Validates if a given object is a EventData instance, and logs corresponding warnings.
+ *
+ * @param {EventData} data event data instance to validate. Precondition: data != undefined
+ * @returns {boolean} Whether the data instance is a valid EventData or not.
+ */
+export function validateEventData(data) {
+  if (!validateEvent(data.eventTypeId, 'splitio-ga-to-split:mapper'))
+    return false;
+
+  if (validateEventValue(data.value, 'splitio-ga-to-split:mapper') === false)
+    return false;
+
+  const { properties } = validateEventProperties(data.properties, 'splitio-ga-to-split:mapper');
+  if (properties === false)
+    return false;
+
+  if (data.timestamp && !isNumber(data.timestamp))
+    return false;
+
+  if (data.key && validateKey(data.key, 'splitio-ga-to-split:mapper') === false)
+    return false;
+
+  if (data.trafficTypeName && validateTrafficType(data.trafficTypeName, 'splitio-ga-to-split:mapper') === false)
+    return false;
+
+  return true;
+}
+
+/**
+ * GaToSplit integration.
+ * This function provides the SplitTracker plugin to ga command queue.
+ *
+ * @param {object} sdkOptions options passed at the SDK integrations settings
+ * @param {object} storage SDK storage passed to track events
+ * @param {object} coreSettings core settings used to define an identity if no one provided as SDK or plugin options
+ */
+function GaToSplit(sdkOptions, storage, coreSettings) {
+
+  const defaultOptions = {
+    filter: defaultFilter,
+    mapper: defaultMapper,
+    // Default eventHandler stores event
+    eventHandler: function (event) {
+      storage.events.track(event);
+    },
```

Comment:
This seems odd, people will not define this in the options. But today if they define something with that name they might override this.